refresh/templates/bluemix/manifest.yml

```diff
@@ -1,23 +1,13 @@
-<% if (Object.keys(services).length > 0) { -%>
-declared-services:
-<% Object.keys(services).forEach(function(serviceType) { -%>
-<% services[serviceType].forEach(function(service) { -%>
-  <%-service.name%>:
-    label: <%-service.label || helpers.getBluemixServiceLabel(serviceType)%>
-    plan: <%-service.plan || 'Lite'%>
-<% }); -%>
-<% }); -%>
-<% } -%>
-<% if (capabilities.autoscale) { -%>
-  <%-capabilities.autoscale%>:
-    label: Auto-scaling
-    plan: free
-<% } -%>
 applications:
-- name: <%-appName%>
-  memory: 128M
-  instances: 1
+- name: <%- bluemix.name || appName %>
+<% if(bluemix.host && bluemix.domain) { -%>
+  host: <%- bluemix.host %>
+  domain: <%- bluemix.domain %>
+<% } else { -%>
   random-route: true
+<% } -%>
+  memory: <%- bluemix.memory || '128M' %>
+  instances: 1
```

Comment:
I think there's a few more pieces we want to be configurable, like disk quota and instances. Are these planned in a future commit?